lib/s3_inventory.rb

```diff
@@ -25,9 +25,12 @@ def initialize(s3_helper, type, preloaded_inventory_file: nil, preloaded_invento
     if type == :upload
       @type = "original"
       @model = Upload
+      @scope = @model.by_users.without_s3_file_missing_verification_status
```

Comment:
The actual change for this file is here. Most are just refactoring changes to make things clearer.